### DISCUSSION TITLE
Stricter certificate validation options

### DISCUSSION THREAD

**Discussion by onne** [webauthn4j/webauthn4j#433] [Ideas]
Hi @ynojima and thank you for this work.
Currently, certificate path trustworthiness validators perform validation with policy qualifiers rejection and revocation checks disabled, statically. Some applications might prefer to require a stricter validation; this can be achieved by allowing to `setPolicyQualifiersRejected` and `setRevocationEnabled` on `CertPathTrustworthinessValidatorBase` concrete implementation instances. Do you have any objection for such change?

**Comment by ynojima** [comment]
Do you really want to have `setRevocationEnabled` option? Most authenticators' attestation certificate doesn't have CRL or OCSP extension and setting `setRevocationEnabled` to `true` just results into error: `CertPathValidatorException: Could not determine revocation status` .
If you would like to enable it, which authenticator do you have in mind?

**Reply by onne** [reply]
We want to allow users to require revocation check by configuration. We understand that some authenticators might suffer from such configuration and we understand that some users might still choose to go that way.
One example we have in mind for a CRL enabled certificate is GlobalSign Root CA - R2, used for android attestations.

**Reply by onne** [reply]
Following up on this. Any comment?

**Comment by ynojima** [comment]
Fixed in #460
